pkg/hubble/relay/server/option.go

```diff
@@ -83,6 +87,22 @@ func WithRetryTimeout(t time.Duration) Option {
 	}
 }
 
+// WithHealthCheckInterval sets the time interval between health checks.
+func WithHealthCheckInterval(t time.Duration) Option {
+	return func(o *options) error {
+		o.healthCheckInterval = t
+		return nil
+	}
+}
```

Comment:
It looks like this is not referenced anywhere, i.e. not hooked up to a command line option. Should it be?

Reply:
Nice catch, I wanted to add a CLI flag for that, but it seems if forgot about it.

But thinking about it, I'm not sure anyone will ever change that interval though. So I'm not sure anymore if we should expose it..

Reply:
Agree. In that case let's drop `options.healthCheckInterval` and `WithHealthCheckInterval` and use `defaults.HealthCheckInterval` in `newHealthServer` directly to make it obvious that the interval cannot be changed.